assets/js/scenes/department/EconClassPackedCircles/econ-class-packed-circles.html

```diff
@@ -1,56 +1,77 @@
 {% load define_action %}
 
-
-{% assign ""|add:source_type|add:" "|add:year as source_text %}
-
-{% assign "Learn more about spending in the "|add:source_type|add:" as PDF" as pdf_text %}
-
-{% assign "Learn more about spending in the "|add:source_type|add:" as Excel" as excel_text %}
-
-{% assign "Dataset Guide for "|add:source_type as guide_text %}
-
-
-<div class="EconClassPackedCirclesSection" id="econ-classification">
-    <h3 class="Title Title--section">Classification of spending items</h3>
-
-    <div class="EconClassPackedCirclesSection-container">
-        <div class="EconClassPackedCirclesSection-item">
-            <div class="EconClassPackedCirclesSection-share u-marginTop10 u-marginBottom15">
-                <div data-component="Share" data-anchor="econ-classification" data-color="{{ color }}"></div>
-            </div>
-
-            <p class="js-tooltips">In addition to exploring the department budget by programmes, the budget is classified according to Economic Classification.</p>
-
-            <p class="js-tooltips">The Economic Classification of a department's expenditure distinguishes between various categories of current expenditure and capital expenditure. The purpose of the economic classification is to categorise transactions according to type of object or input, such as <em>compensation of employees</em> or <em>capital assets</em>.</p>
-
-            <p class="js-tooltips">This visualisation highlights how many categories of expenditure there are for this department at Economic Classification level 4, and which ones are the biggest.</p>
-
-            <ul class="LinksList">
+        {% assign ""|add:source_type|add:" "|add:year as source_text %}
+        
+        {% assign "Learn more about spending in the "|add:source_type|add:" as PDF" as pdf_text %}
+        
+        {% assign "Learn more about spending in the "|add:source_type|add:" as Excel" as excel_text %}
+        
+        {% assign "Dataset Guide for "|add:source_type as guide_text %}
+        
+        
+        <div class="EconClassPackedCirclesSection" id="econ-classification">
+            <h3 class="Title Title--section">Classification of spending items</h3>
+        
+            <div class="EconClassPackedCirclesSection-container">
+                <div class="EconClassPackedCirclesSection-item">
+                    <div class="EconClassPackedCirclesSection-share u-marginTop10 u-marginBottom15">
+                        <div data-component="Share" data-anchor="econ-classification" data-color="{{ color }}"></div>
+                    </div>
+            {% if prog_econ4_circles_data.get_dataset != None %}
```

Comment:
```suggestion
            {% if prog_econ4_circles_data.get_dataset %}
```

We can treat the none as "falsy" which means we only need `if prog_econ4_circles_data.get_dataset` - we don't need to check if it's precisely `None`

Reply:
we don't need this condition because the specific items already have conditionals